Sources/App/Commands/ReAnalyzeVersions.swift

```diff
@@ -54,51 +54,48 @@ enum ReAnalyzeVersions {
                     try await reAnalyzeVersions(
```

Comment:
Needs

```
            Current.setLogger(logger)
```

after `let logger = Logger(component: "re-analyze-versions")` on line 49.

Essentially, we need to make sure that every `Logger` initialiser `Logger(component: ...)` is followed by a `Current.setLogger(logger)`.